op-guide/binary-deployment.md

```diff
@@ -0,0 +1,486 @@
+| min_free_kbytes | The setting for `vm.min_free_kbytes` in `sysctl.conf` needs to be high enough |
+| User Open Files Limit | For database administrators, set the number of TiDB open files to `1000000` |
+| System Open File Limits | Set the number of system open files to `1000000` |
+| User Process Limits | For TiDB users, set the `nproc` value to `4096` in `limits.conf` |
+| Address Space Limits | For TiDB users, set the space to `unlimited` in `limits.conf` |
+| File Size Limits | For TiDB users, set the `fsize` value to `unlimited` in `limits.conf` |
+| Disk Readahead | Set the value of the `readahead` data disk to `4096` at a minimum |
+| NTP service | Configure the NTP time synchronization service for each node |
+| SELinux  | Turn off the SELinux service for each node |
+| CPU Frequency Scaling | It is recommended to turn on CPU overclocking |
+| Transparent Hugepages | For Red Hat 7+ and CentOS 7+ systems, it is required to set the Transparent Hugepages to `always` |
+| I/O Scheduler | Set the I/O Scheduler of data disks to the `deadline` mode |
+| vm.swappiness | Set `vm.swappiness = 0` |
+
+
+> **Note**: To adjust the operating system parameters, contact your system administrator.
+
+### Database running user
+
+| Configuration | Description |
+| :-- | :---------------------------- |
+| LANG environment | Set `LANG = en_US.UTF8` |
+| TZ time zone | Set the TZ time zone of all nodes to the same value |
+
+## Create the database running user account
+
+In the Linux environment, create TiDB on each installation node as a database running user, and set up the SSH mutual trust between cluster nodes. To create a running user and open SSH mutual trust, contact the system administrator. Here is an example:
+
+```bash
+$ useradd tidb
+$ usermod -a -G tidb tidb
+$ su - tidb
+Last login: Tue Aug 22 12:06:23 CST 2017 on pts/2
+-bash-4.2$ ssh-keygen -t rsa
+Generating public/private rsa key pair.
+Enter file in which to save the key (/home/tidb/.ssh/id_rsa):
+Created directory '/home/tidb/.ssh'.
+Enter passphrase (empty for no passphrase):
+Enter same passphrase again:
+Your identification has been saved in /home/tidb/.ssh/id_rsa.
+Your public key has been saved in /home/tidb/.ssh/id_rsa.pub.
+The key fingerprint is:
+5a:00:e6:df:9e:40:25:2c:2d:e2:6e:ee:74:c6:c3:c1 tidb@t001
+The key's randomart image is:
++--[ RSA 2048]----+
+|    oo. .        |
+|  .oo.oo         |
+| . ..oo          |
+|  .. o o         |
+| .  E o S        |
+|  oo . = .       |
+| o. * . o        |
+| ..o .           |
+| ..              |
++-----------------+
+
+-bash-4.2$ cd .ssh
+-bash-4.2$ cat id_rsa.pub >> authorized_keys
+-bash-4.2$ chmod 644 authorized_keys
+-bash-4.2$ ssh-copy-id -i ~/.ssh/id_rsa.pub 192.168.1.100
+```
+
+## Download the official binary package
+
+TiDB provides the official binary installation package that supports Linux. For the operating system, it is recommended to use Redhat 7.3+, CentOS 7.3+ and higher versions.
```

Comment:
```suggestion
```

I moved the part "TiDB provides the official binary installation package that supports Linux." to the overview.  The part about Operating System moved under the "Operating System header"